portal-api/src/drivers/fileshare.js

```diff
@@ -10,16 +10,12 @@ const setConfig = (fileshareConfig) => {
 };
 
 const getConfig = (fileshare = 'portal') => {
-  const config = fileshare === 'workflow'
-    ? AZURE_WORKFLOW_FILESHARE_CONFIG
-    : AZURE_PORTAL_FILESHARE_CONFIG;
+  const config = fileshare === 'workflow' ? AZURE_WORKFLOW_FILESHARE_CONFIG : AZURE_PORTAL_FILESHARE_CONFIG;
```

Comment:
`workflow` from constant please?

Reply:
this was not changed/introduced as part of the PR. I'd like to keep the scope focused on eStore changes